packages/ai/core/tool/mcp/mcp-sse-transport.ts

```diff
@@ -30,10 +32,10 @@ export class SseMCPTransport implements MCPTransport {
 
       const establishConnection = async () => {
         try {
+          const headers = new Headers({...this.headers});
```

Comment:
tiny nit! the spread isn't necessary 

```suggestion
          const headers = new Headers(this.headers);
```